utils.py

```diff
@@ -0,0 +1,114 @@
+from files import tasks
+from decorators import view_decorator, create_decorator, timer_decorator
+
+def gen_id(tasks):
+    if not len(tasks):
+        return 1
+    return max(tasks.keys()) + 1
+
+def save_task(tasks):
+    with open("tasks.txt", "w") as file:
+        file.write(tasks_to_strings(tasks))
+
+def tasks_to_strings(tasks):
+    result = ""
+    for task_id, task_values in tasks.items():
+        result += f"{task_id}: {task_values['title']} | {task_values['desc']} | {task_values['priority']} | {task_values['status']}\n"
+    return result
+
+def priority_def():
+    try:
+           while True:
+            priority = int(input("Выберите приоритет задачи (1 - low, 2 - middle, 3 - high): "))
+            if priority == 1:
```

Comment:
And you spend more time placing **all logic** in the `try' block.
In fact, this is only needed to catch the "potential" exception on line 22.

You can convert this as:
```python
def get_priority() -> str:
    map = {1: "low", 2: "medium", 3: "high"}
    user_choice = 0
    while user_choice is not in map:
        try:
            user_choice = int(input(...))
        except ValueError:
            ...
    
    return map[user_choice]
```

Pay attention to the dictionary. When you need to **map** some value to another, this is a good data structure to succeed.

Reply:
Add you can slightly modify the snippet above to pass the mapping as an argument.
Looking for some value in the map collection is a common task for this "program".